### PR TITLE
Model location spec

### PR DESCRIPTION
**Pull request by RichardLaskowski**
#2 
#3 
#4
#5 
#6 
#7

### CODE REVIEW

**Review by cdimoulis — CHANGES_REQUESTED** [review]
@RichardLaskowski Looks like this has code that satisfies other issues like #6 Check out the code that is included in this PR and if there are any other completed issues and tag these also. 

However if work still needs to be done on other tickets then don't tag them and we'll do a new branch and PR for those.

**Comment by RichardLaskowski** [comment]
#3 
#4 
#5 
#6 
#7
This branch includes the previous issue due to being branched from the completed event-model-spec branch

File: app/models/core/event.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Core::Event < ApplicationRecord
+     belongs_to :location, optional: true
+     validates :title, presence: :true
+     :location_id
```
Comment (cdimoulis) [inline app/models/core/event.rb]: Didn't we remove this? Just putting there here is like placing a string on a line of code.
Was this potentially in a different branch?

File: app/models/core/location.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class Core::Location < ApplicationRecord
+    has_many :events
+    validates :name, presence: :true
+    :location_id
```
Comment (cdimoulis) [inline app/models/core/location.rb]: Didn't we remove this? Different branch? Needs to be removed

File: spec/models/core/location_spec.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+RSpec.describe Core::Location, type: :model do
+
+  context 'associations' do
+    it {is_expected.to have_many(:events)}
+  end
+
+  context '.save' do
+    it 'succeeds' do
+      test_location = build(:core_location)
+      expect(test_location.save).to be_truthy
```
Comment (cdimoulis) [inline spec/models/core/location_spec.rb]: change to `expect(text_location.save).to be(true)`. This function returns true if successful. Of course run the test after making the change and committing.

**Review by cdimoulis — APPROVED** [review]

**Pull request merged**